Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -797,6 +825,61 @@ AXTextMarker AXTextMarker::findMarker(AXDirection direction, AXTextUnit textUnit
         }
         return linePosition;
     }
+
+    if (textUnit == AXTextUnit::Word) {
+        unsigned offset = this->offset();
+        AXTextMarker resultMarker = *this;
+
+        String flattenedRuns = currentRuns->substring(0, currentRuns->totalLength());
```

Comment:
Feels like `AXTextRuns` should have a method called `toString()` or something.

```cpp
// AXTextRuns.h
struct AXTextRuns {

String substring(unsigned start, unsigned length = StringImpl::MaxLength) const;
String toString() const { return substring(0); }
};
```

Reply:
Oh yeah, let's add that